src/reducers/arcReducer.ts

```diff
@@ -450,12 +453,9 @@ const arcReducer = (state = initialState, action: any) => {
             [avatarAddress] : {
               members: {
                 [accountAddress]: {
-                  $merge : payload.beneficiary,
+                  $merge: payload.beneficiary,
                   // remove pending redemptions from this account
-                  redemptions :
-                    payload.redemptions ?
-                      { [proposalId]: {$merge: payload.redemptions}} :
-                      { $unset: [proposalId] },
+                  redemptions: { $unset: [proposalId] },
```

Comment:
couldn't there still be redemptions left, if the DAO doesnt have enough funds to cover them all? I think we still want it the way it was

Reply:
I tried to update redemptions on the redeem event via `getRedemptions`, but it seems to return old values.

Reply:
hmm, so we will still end up with the weird row disappearing then reappearing... we may have to manually figure out which ones are being redeemed

Reply:
i'll try to fix that